imagegw/shifter_imagegw/imagemngr.py

```diff
@@ -568,8 +574,8 @@ def pull(self, session, image, testmode=0):
             self.logger.debug("Creating New Pull Record")
             rec = self.new_pull_record(request)
             ident = rec['_id']
-            self.logger.debug("ENQUEUEING Request")
-            self.update_mongo_state(ident, 'ENQUEUED')
+            self.logger.debug("PENDING Request")
+            self.update_mongo_state(ident, 'PENDING')
```

Comment:
will this change of state name from ENQUEUED to PENDING generate any manual cleanup an administrator might need to do?